### PR TITLE
[config] use YAML over INI file for config. Automatically create local config interactively

### PR DESCRIPTION
**Pull request by seferov**
A suggestion:
* Use YAML instead of INI.
* Create local config interactively by running composer install

### CODE REVIEW

File: web/github_hook.php [hunk 1]
```diff
@@ -7,10 +7,18 @@
...
+require $app_root . '/vendor/autoload.php';
+
 // Git variables
 $branch = 'master';
 $header = 'HTTP_X_HUB_SIGNATURE';
-$secret = parse_ini_file($app_root . '/app/config/config.ini')['github_key'];
+
+$config_file = $app_root . '/app/config/config.yml';
+if (!file_exists($config_file)) {
+    throw new \InvalidArgumentException(sprintf('File %s is missing. Run composer install.', $config_file));
```
Comment (flodolo) [inline web/github_hook.php]: A couple of nits here:
* We keep one space between the not and condition, i.e. `(! file_exists($config_file))`.
* We tend to use print/echo over printf, so `"File {$config_file} is missing. Run composer install."`.

**Review by flodolo — CHANGES_REQUESTED** [review]
Thanks, this looks interesting, and I could port this approach to other projects with a bit of testing.

I need to test it locally, but it should work.

**Comment by seferov** [comment]
@flodolo I updated the code as you suggested.
I tested it locally and it is working fine. If it is ok and after your test I can port it other projects

**Review by flodolo — APPROVED** [review]
Tested locally and it works great. I'll merge later because the automation is probably going to get stuck on the missing parameter

**Pull request merged**